nvme.c

```diff
@@ -9439,16 +9480,18 @@ static int tls_key(int argc, char **argv, struct command *command, struct plugin
 	const char *keyfile = "File for list of keys.";
 	const char *import = "Import all keys into the keyring.";
 	const char *export = "Export all keys from the keyring.";
+	const char *remove = "Remove key from the keyring.";
 
 	FILE *fd;
```

Comment:
`FILE` type can be changed to `_cleanup_file_ int` type.

Reply:
No, it's the wrong type, the cleanup expect an filedescriptor not a stream.

Reply:
I thought just to use the cleanup function instead of the free with the goto label out and also the file descriptor can be used instead of the stream (by using `read()`, `write()` or `fdopen()` etc.) but understood as not good as complicated. By the way seems a file stream cleanup function can be created instead I thought.

Reply:
Yep, I was also thinking along this line, was just to lazy.